acceptancetests/assess_network_health.py

```diff
@@ -502,6 +708,16 @@ def is_ipv6(self, address):
             return False
         return True
 
+    def ip_in_cidr(self, address, cidr):
```

Comment:
This should be a standalone function, there is no need for it to be a member method.